df.entities.xml

```diff
@@ -482,8 +482,7 @@
                 <enum base-type='int16_t' type-name='ethic_response'/>
             </static-array>
 
-            <static-array name='values' type-name='int32_t' count='32' index-enum='value_type'/>
-            <static-array name='values_2' type-name='int32_t' count='32' index-enum='value_type' since='v0.42.01'/>
+            <static-array name='values' type-name='int32_t' count='64' index-enum='value_type'/>
```

Comment:
Yes, there are still 64 elements. In my test world, `values[33]` and beyond are 0 for all historical entities. Toady apparently reserved space for 64 value types but only defined 33.